ebsco/edsapi-koha-plugin
# Change .pl file permissions to 755 so Apache can execute

At the moment, you have to reset the permissions on the .pl
files to 755 every time you upload a plugin to Koha, because
Apache can't execute the files.

If we change them in the git repository, they should be
correct when they're uploaded and processed by Koha.

_TEST PLAN_

Before applying:

1) Create a .kpz package and upload to Koha
2) Do a search
3) Note that you get a 500 error

4) Apply the patch
5) Build a new .kpz package and upload to Koha
6) Do a search
7) Note that your search works correctly now
